nautobot/extras/migrations/0054_role_and_alter_status.py

```diff
@@ -0,0 +1,63 @@
+# Generated by Django 3.2.16 on 2022-11-19 23:15
+
+import django.core.serializers.json
+from django.db import migrations, models
+import nautobot.core.fields
+import nautobot.extras.models.mixins
+import nautobot.extras.utils
+import nautobot.utilities.fields
+import uuid
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("contenttypes", "0002_remove_content_type_name"),
+        ("extras", "0053_relationship_required_on"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="status",
+            name="name",
+            field=models.CharField(max_length=100, unique=True),
```

Comment:
Why isn't there also a migration on `status.slug`, which is also changing max_length?

Reply:
migration file 0014_auto_slug shows slugs' `max_length` has been 100.

Reply:
Huh, that seems like a bug in our `AutoSlugField` class then, since Status declares:

```python
    slug = AutoSlugField(populate_from="name", max_length=50)
```

Out of scope for this PR though.